activerecord/lib/active_record/persistence.rb

```diff
@@ -254,8 +254,8 @@ def upsert(attributes, returning: nil, unique_by: nil)
       #     { title: 'Clean Code', author: 'Robert', isbn: '2' }
       #   ], unique_by: { columns: %w[ isbn ] })
       #
-      def upsert_all(attributes, returning: nil, unique_by: nil)
-        InsertAll.new(self, attributes, on_duplicate: :update, returning: returning, unique_by: unique_by).execute
+      def upsert_all(attributes, returning: nil, unique_by: nil, updatable_columns: nil)
+        InsertAll.new(self, attributes, on_duplicate: :update, returning: returning, unique_by: unique_by, updatable_columns: updatable_columns).execute
```

Comment:
For the name of the new keywords, though, `updatable_columns:` seems a bit verbose.

I'd suggest, simply, `:update`.

More conversation here https://github.com/rails/rails/pull/35077#issuecomment-471826877 and starting here https://github.com/rails/rails/pull/35636#issuecomment-473481908